### PR TITLE
Use space uuid's instead of space id's

### PR DESCRIPTION
**Pull request by willot**
## Issue
Resolves first part of Issue14 

## What was done
- [x] Add the UUID to every object in the back end and front end 

## How to test
1. Everything should be the same

### FOR DEVS
Feature URL: See feature branch

### FOR APPROVAL
Approval URL: See Approval branch

### CODE REVIEW

File: api/src/test/kotlin/com/ford/internalprojects/peoplemover/assignment/AssignmentControllerInTimeApiTest.kt [hunk 1]
```diff
@@ -288,7 +298,7 @@ class AssignmentControllerInTimeApiTest {
     @Test
     fun `POST should return 403 if user does not write access`() {
 
-        val createAssignmentsRequest = CreateAssignmentsRequest(LocalDate.now(), Person("", -9999), HashSet())
+        val createAssignmentsRequest = CreateAssignmentsRequest(LocalDate.now(), Person(name ="", spaceUuid = "-9999", spaceId = -9999), HashSet())
```
Comment (egriff38ford) [inline api/src/test/kotlin/com/ford/internalprojects/peoplemover/assignment/AssignmentControllerInTimeApiTest.kt:301]: Could probably look like an actual UUID

File: api/src/test/kotlin/com/ford/internalprojects/peoplemover/product/ProductControllerApiTest.kt [hunk 2]
```diff
@@ -44,6 +44,7 @@ import org.springframework.test.context.junit4.SpringRunner
 import org.springframework.test.web.servlet.MockMvc
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import java.util.*
```
Comment (egriff38ford) [inline api/src/test/kotlin/com/ford/internalprojects/peoplemover/product/ProductControllerApiTest.kt:47]: ?

Comment (willot) [reply api/src/test/kotlin/com/ford/internalprojects/peoplemover/product/ProductControllerApiTest.kt:47]: It is not in our code on our machine........................................................................................................................................................................................................................................................................

Comment (willot) [reply api/src/test/kotlin/com/ford/internalprojects/peoplemover/product/ProductControllerApiTest.kt:47]: we corrected the other one that were really in the code base

**Review by egriff38ford — APPROVED** [review]
Minor nits, but overall: 🚢 🇮🇹

**Pull request merged**